### ISSUE TITLE
[SUPPORT] Huawey changed the file format and hitrava cannot convert it any longer

### ISSUE THREAD

**Issue by jimiend** [CTHRU/Hitrava#78]
this is the content of the zip file
<img width="452" alt="Screenshot 2024-09-12 at 17 11 58" src="https://github.com/user-attachments/assets/5a6dbdc8-d6f0-4dcd-8f00-079ebfdc746b">

Here the motion detail directory
<img width="452" alt="Screenshot 2024-09-12 at 17 13 04" src="https://github.com/user-attachments/assets/e3291710-6657-4338-932f-785bd3649fd1">

I'm attaching one file for reference
[motion path detail data1724429219003.json](https://github.com/user-attachments/files/16981803/motion.path.detail.data1724429219003.json)

and below the error when I try to run the script on it
```
dpacific@DPACIFIC-M-V63K ~ % cd expect/Hitrava-master
dpacific@DPACIFIC-M-V63K Hitrava-master % python3 Hitrava.py -j input/motion\ path\ detail\ data1724429219003.json
2024-09-12 17:15:49,018 - INFO - main - Hitrava version 5.1.2 (build 2308.2601) started with arguments ['-j', 'input/motion path detail data1724429219003.json']
2024-09-12 17:15:49,018 - INFO - main - Running on Python version 3.9.12
2024-09-12 17:15:49,020 - INFO - parse - Found one or more activities in JSON at index 0 to parse from 2023-08-01 (YYY-MM-DD)
2024-09-12 17:15:49,020 - ERROR - parse - Error parsing JSON file <input/motion path detail data1724429219003.json>
'attribute'
Traceback (most recent call last):
  File "/Users/dpacific/expect/Hitrava-master/Hitrava.py", line 1285, in parse
    hi_activity = self._parse_activity(activity_dict)
  File "/Users/dpacific/expect/Hitrava-master/Hitrava.py", line 1304, in _parse_activity
    hitrack_data = activity_dict['attribute']
KeyError: 'attribute'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/dpacific/expect/Hitrava-master/Hitrava.py", line 2170, in <module>
    main()
  File "/Users/dpacific/expect/Hitrava-master/Hitrava.py", line 2156, in main
    hi_activity_list = hi_json.parse(args.from_date)
  File "/Users/dpacific/expect/Hitrava-master/Hitrava.py", line 1300, in parse
    raise Exception('Error parsing JSON file <%s>', self.json_file.name)
Exception: ('Error parsing JSON file <%s>', 'input/motion path detail data1724429219003.json')
```

**Comment by CTHRU** [comment]
Well, unfortunately it is a known issue, see #77.

There is nothing much that can be done about it as the detailed track data is simply gone. Unless Huawei is willing to put it back in some form or other, I think you would have to rely on other solutions. In any case, I found it strange that the data disappeared, because for sure they have it since it is displayed in the app. Maybe someone made a mistake on their part and removed the data export? You could always try to contact them and mention your GDPR right to access the data if you are in Europe.

**Comment by jimiend** [comment]
It was a long discussion with Aspigel (GDPR legal entity for Huawei in Europe)
I also needed to file a formal complaint to the GDPR authority
right now the json formatting is back as it was but the activities are spread across multiple files

do not hesitate to reach out if you face the same issue and you are based in Europe

**Comment by CTHRU** [comment]
> It was a long discussion with Aspigel (GDPR legal entity for Huawei in Europe) I also needed to file a formal complaint to the GDPR authority right now the json formatting is back as it was but the activities are spread across multiple files
> 
> do not hesitate to reach out if you face the same issue and you are based in Europe

Wow! That is great news. And thank you so much for taking the effort to get matters resolved with Aspiegel / GDPR. I will check in the coming days/weeks whether the data is back and in which form. I keep you posted.

**Comment by CTHRU** [comment]
@jimiend I can confirm that the required data seems to be back after I requested my data again from Huawei. I will work on a solution to get Hitrava up and running again with the format changes.

**Comment by CTHRU** [comment]
Closed in version 6.0.0 (#80)
